test/goHome.test.ts

```diff
@@ -0,0 +1,102 @@
+import { chromium, Page, Browser, BrowserContext, Cookie } from "playwright"
+import { createCookieIfDoesntExist } from "../src/common/util"
+import { hash } from "../src/node/util"
+
+async function setTimeoutPromise(milliseconds: number): Promise<void> {
+  return new Promise((resolve, _) => {
+    setTimeout(() => {
+      resolve()
+    }, milliseconds)
+  })
+}
+
+describe("login", () => {
+  let browser: Browser
+  let page: Page
+  let context: BrowserContext
+
+  beforeAll(async () => {
+    browser = await chromium.launch()
+    // Create a new context with the saved storage state
+    const storageState = JSON.parse(process.env.STORAGE || "")
```

Comment:
I think `JSON.parse("")` will error.

```suggestion
    const storageState = JSON.parse(process.env.STORAGE || "{}")
```